### ISSUE TITLE
[KEYBOARD]: Focus is being moved to search results when users zoom their page with browser zoom

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#492]
## Issue
Focus is being moved to the search results when keyboard users zoom their page using `CMD/CTRL + Plus`. This was noted as an SC 3.3.2 violation. Steps to recreate below.

## Related Issues
* https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/490
* https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/491

## Audit Finding
* Note 10, Defect 1 of 1

## Acceptance Criteria
[ ] Current keyboard focus is retained when keyboard user zooms the page in or out using browser zoom feature. 
[ ] The number of results should update as before, but the yellow focus halo should stay where it was previous.
[ ] Run automated scans periodically during development
[ ] Run the axe-core plugin while developing new components. Many automated errors can be avoided or fixed during the development cycle.
[ ] Internal peer review 
[ ] After peer review, share with the FL team for Product/Design sign off and/or FE Review Team 
[ ]  Documentation and points of contact for any new backend dependencies
[ ]  Links to important dashboards for investigating relevant issues

## Definition of done
[ ] Events are logged appropriately
[ ] Documentation has been updated, if applicable
[ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
[ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/

## Steps to Recreate
1. Load https://staging.va.gov/find-locations/
2. Enter a search like "Austin, TX"
3. Press the Search button
4. Verify focus has moved to the search results (yellow halo). This is intended.
5. Press `SHIFT + TAB` a few times to move back to the form
5. Press `CMD + Plus` on Mac or `CTRL + Plus` on Windows to zoom the browser in
6. Verify focus has again been moved to the search results

There may be some edge cases to work out, because if a user has tabbed into the search results and zoomed their browser, we are currently rerunning the search. I don't think that's quite right since the map is not changing, just being increased or decreased in size. We should remember where users' focus was in this case, and not re-render our search results.

## WCAG or Vendor Guidance (optional)
* [On Input: Understanding SC 3.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html)

**Comment by mmiddaugh** [comment]
@brangi to evaluate this situation

**Comment by mmiddaugh** [comment]
Closing this issue as the acceptance criteria has been added to [#4324].
